app/data/1_0001.json

```diff
@@ -5130,25 +5130,19 @@
                         }
                     ],
                     "type": "questionnaire"
-                }
-            ],
-            "id": "general-information",
-            "title": "General Information"
-        },
-        {
-            "blocks": [
+                },
                 {
                     "id": "ready-to-submit-completed",
                     "sections": [
                         {
-                            "description": "<p>Submission</p><p>You will not be able to access or change your answers on submitting the questionnaire</p><p>If you wish to review your answers please select the relevant Completed sections</p><p>If you do not submit your questionnaire, your responses will be collected when the online questionnaire closes</p>",
+                            "description": "",
                             "id": "ready-to-submit-completed-section",
                             "questions": [
                                 {
                                     "answers": [],
-                                    "description": "",
+                                    "description": "<ul><li><p>You will not be able to access or change your answers on submitting the questionnaire</p></li><li><p>If you wish to review your answers please select the relevant Completed sections</p></li><li><p>If you do not submit your questionnaire, your responses will be collected when the online questionnaire closes</p></li></ul>",
```

Comment:
I think this is okay for now. As a rule, I think we're trying not to introduce HTML tags into the schema. However, the `description` key does not support the `list` construct (in the same way as `guidance`, for example).